demo_openvino.py

```diff
@@ -12,238 +12,14 @@
 import matplotlib.pyplot as plt
 import cv2
 import numpy as np
-
+import model as m
```

Comment:
One letter names is a bad practice (there are some exceptions, such as cycle iterators), because the are meaningless and make code hard to read and understand.